q2_assembly/quast/quast.py

```diff
@@ -227,6 +236,14 @@ def _zip_additional_reports(path_to_dirs: list, output_filename: str) -> None:
             _zip_dir(zipf, directory)
 
 
+def _move_references(src, dest):
+    file_patterns = ["*.fasta", "*.fa"]
+    os.makedirs(os.path.join(dest, "quast_downloaded_references"), exist_ok=True)
+    for pattern in file_patterns:
+        for fp in glob.glob(os.path.join(src, pattern)):
+            shutil.copy(fp, os.path.join(dest, "quast_downloaded_references"))
```

Comment:
This loop is a bit redundant, given the two patterns you have here: if you have a `*.fa` pattern - it should already contain all the `*.fasta` files so there should be no need to loop through both.